Analyses/fcl/DSBFieldDump.fcl

```diff
@@ -11,6 +11,7 @@ services : {
   GlobalConstantsService  : { inputFile      : "Offline/GlobalConstantsService/data/globalConstants_01.txt" }
     GeometryService	    : {
       inputFile      : "Offline/Mu2eG4/geom/geom_common.txt"
+      bFieldFile          : "Offline/Mu2eG4/geom/bfgeom_v01.txt"
```

Comment:
Given the override below, shouldn't this be
Offline/Mu2eG4/geom/bfgeom_reco_v01.txt